content/vendor/packaging/private-images.md

```diff
@@ -67,6 +71,27 @@ However, the same secret will be added to those PodSpecs as well.
 
 > KOTS [Application](/reference/v1beta1/application/) deployments are supported via image tags in all use cases. KOTS has limited support for deploying via image digests. Use of image digests are only supported for fully online installs where all images can be pulled from the Replicated registry, a public repo, or proxied from a private repo via the Replicated registry.
 
+## Replicated Private Registry
+
+When using the Replicated Private Registry, you have 2 options to connect with the `registry.replicated.com` container registry
```

Comment:
how about a colon after "registry"?

"When using the Replicated Private Registry, you have 2 options to connect with the `registry.replicated.com` container registry:"